apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/PublisherClient.cs

```diff
@@ -407,6 +418,17 @@ internal void Validate()
         /// <returns>A <see cref="Task"/> that completes when all queued messages have been published; or cancels if
         /// <paramref name="timeout"/> expires.</returns>
         public virtual Task ShutdownAsync(TimeSpan timeout) => ShutdownAsync(new CancellationTokenSource(timeout).Token);
+
+        /// <summary>
+        /// Disposes this <see cref="PublisherClient"/> asynchronously.
+        /// </summary>
+        /// <remarks>
+        /// This method asynchronously waits for the time interval as specified in the <see cref="Settings.DisposeTimeout"/>
+        /// for the <see cref="PublisherClient"/> to send any pending messages before aborting the
+        /// clean shutdown process, that may leave some locally queued messages unsent.
+        /// The time interval can be customized by setting the <see cref="Settings.DisposeTimeout"/>.
```

Comment:
```suggestion
        /// This method asynchronously waits for the time interval as specified in the <see cref="Settings.DisposeTimeout"/>
        /// for the <see cref="PublisherClient"/> to send any pending messages. If the clean shutdown is not
        /// complete after this time, it is aborted; this may leave some locally queued messages unsent.
        /// The time interval can be customized by setting the <see cref="Settings.DisposeTimeout"/>.
```